Rails/subscribe.md

```diff
@@ -0,0 +1,4 @@
+Name | URL
```

Comment:
subscribe.md?? 

These 2 links should be added in "Useful_links" file, same as what we have done in Android newsletter, please check https://github.com/GDGAhmedabad/Learning-Resources/blob/master/android/Useful_links.md

And please delete this Subscribe file.